Results reviewer request part1 (#82)
greenelab/annorxiver_manuscript: content/04.results.md

We plotted the 95% confidence interval for each reported token.
**E.** The tokens with the largest absolute differences in abundance appear to be stylistic.
](https://raw.githubusercontent.com/danich1/annorxiver/b8d20a1e7f59e43817cfdb8a5b0156b607b2c900/figure_generation/output/figure_one_panels.png){#fig:corpora_comparison_panels width="100%"}

Documents within bioRxiv were slightly longer than those within PMCOA, but both were much longer than those from the control (NYTAC) (Table {@tbl:corpora_stats}).
The average sentence length, the fraction of pronouns, and the use of the passive voice were all more similar between bioRxiv and PMC than they were to NYTAC(Table {@tbl:corpora_stats}).
The Kullback–Leibler (KL) divergence of term frequency distributions between bioRxiv and PMCOA were low, especially among the top few hundred tokens (Figure {@fig:corpora_comparison_panels}A).
As more tokens were incorporated, the KL divergence started to increase but remained much lower than the biomedical corpora compared against NYTAC.
These findings support our notion that bioRxiv is linguistically similar to the PMCOA repository.

> Terms like "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B, {@fig:corpora_comparison_panels}C and Supplementary Figure {@fig:corpora_comparison_panels_bio_v_pmid}).

Reviewer: ```suggestion
The terms "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B, {@fig:corpora_comparison_panels}C and Supplementary Figure {@fig:corpora_comparison_panels_bio_v_pmid}).
```

When controlling for the differences in the body of documents to identify textual changes associated with the publication process, we found that tokens such as "et" "al" were enriched for bioRxiv while "$\pm$", "–" were enriched for PMCOA (Figure {@fig:corpora_comparison_panels}D, {@fig:corpora_comparison_panels}E). 

> When removing special and single-character tokens, more figure-related terms appeared enriched for published articles, and more research-related articles appeared enriched for bioRxiv (Supplementary Figure {@fig:corpora_comparison_panels_pre_v_pub}).

Reviewer: What is a "research-related article"? I am missing something. Do you mean terms associated with research vs terms associated with presentation (i.e., fig vs figure)? I am a bit confused here.

Author: >What is a "research-related article"? I am missing something. Do you mean terms associated with research vs terms associated with presentation (i.e., fig vs figure)? I am a bit confused here.

I meant to say terms related to research rather journal presentation. Will update this sentence to be more clear.

Furthermore, we found that specific changes appeared to be related to journal styles: "figure" was more common in bioRxiv while "fig" was relatively more common in PMCOA.
Other changes appeared to be associated with an increasing reference to content external to the manuscript itself: the tokens "supplementary", "additional" and "file" were all more common in PMCOA than bioRxiv, suggesting that journals are not simply replacing one token with another but that there are more mentions of such content after peer review.

These results suggest that the text structure within preprints on bioRxiv is similar to published articles within PMCOA.
The differences in uptake across fields are supported by the authors' categorization of their articles and the text within the articles themselves.
At the level of individual manuscripts, the most change terms appear to be associated with typesetting, journal style, and an increasing reliance on additional materials after peer review.

> Following our global analysis, we performed principal component analysis on document emebddings generated by the bioRxiv corpus to inspect latent concepts captured within them.

Reviewer: ```suggestion
Following our analysis of tokens, we examined the principal components of document embeddings derived from bioRxiv.
```

> Our analysis revealed that the strongest prinicpal component (PC) separated cell biology and informatics-related fields, while the second strongest component separated bioinformatics concepts from neuroscience concepts.

Reviewer: ```suggestion
We found that the top principal components separated methodological approaches and research fields.
Preprints from certain topic areas that spanned approaches from informatics-related to cell biology could be distinguished using these principal components (see Supplementary Results).
```

> We explore these results further within our supplemental section (see 'Document embeddings derived from bioRxiv reveal fields and subfields' for more information).

Reviewer: ```suggestion
```
